app/assets/javascripts/mtl/clickables.coffee

```diff
@@ -1,3 +1,16 @@
+$.rails.href = (element) ->
+  element.data('mtl-href') || element[0].href
+
+$(document).on 'click.rails', '[data-confirm]:not(a), [data-method]:not(a)', (e) ->
```

Comment:
shouldn't we merge this with [data-mtl-href], i.e. why don't we handle all mtl-href through this too?